src/stavetempo.ts

```diff
@@ -20,107 +18,52 @@ export class StaveTempo extends StaveModifier {
     return Category.StaveTempo;
   }
 
-  protected tempo: StaveTempoOptions;
-  protected shiftX: number;
-  protected shiftY: number;
-  /** Font size for note. */
-  public renderOptions = { glyphFontScale: 30 };
-
   constructor(tempo: StaveTempoOptions, x: number, shiftY: number) {
     super();
 
-    this.tempo = tempo;
+    this.setTempo(tempo);
     this.position = StaveModifierPosition.ABOVE;
     this.x = x;
-    this.shiftX = 10;
-    this.shiftY = shiftY;
-  }
-
-  setTempo(tempo: StaveTempoOptions): this {
-    this.tempo = tempo;
-    return this;
+    this.setXShift(10);
+    this.setYShift(shiftY);
   }
 
-  setShiftX(x: number): this {
-    this.shiftX = x;
-    return this;
-  }
+  private durationToCode: Record<string, string> = {
```

Comment:
Should we continue to use the JS prefix for private members, i.e. `#durationToCode`?

Reply:
done